enrollment-server/pom.xml

```diff
@@ -168,6 +168,13 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-deploy-plugin</artifactId>
+                <configuration>
+                    <skip>true</skip>
```

Comment:
Or just use a shorter way and define the maven property `maven.deploy.skip` https://maven.apache.org/plugins/maven-deploy-plugin/deploy-mojo.html#skip

Reply:
It's ok this way for consistency reasons.